bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -72,6 +72,13 @@ spec:
                 exit 0
             }
             trap quit SIGTERM
+
+            # Enable OVS hardware offload
```

Comment:
From OCP 4.6, ovs runs in systemd instead of the container. So this logic will not be invoked.